### PR TITLE
INT-221: Redlink dialog isn't firing under certain [currently unknown] circumstances

### PR DESCRIPTION
**Pull request by mari-mcmurtrie**
The cause was due to URL-encoded link title. So if English, it worked but if Japanese, then not. Fixed to decode URL-encoded title at the right spot. Ticket: https://wikia-inc.atlassian.net/browse/INT-221

### CODE REVIEW

**Comment by mari-mcmurtrie** [comment]
@inez will you CR? The reason I ask you is simply because you are the one last touched this file. Thanks!

File: extensions/wikia/CreatePage/js/CreatePage.js [hunk 1]
```diff
@@ -263,7 +263,7 @@ var CreatePage = {
 
 	redLinkClick: function( e, titleText ) {
 		'use strict';
-		var title = new mw.Title.newFromText( titleText ),
+		var title = new mw.Title.newFromText( decodeURIComponent( titleText ) ),
```
Comment (RafLeszczynski) [inline extensions/wikia/CreatePage/js/CreatePage.js:266]: @inez, @mari-mcmurtrie: is the a case where `titleText` could be `undefined`? If yes, are we OK with `decodeURIComponent` throwing `Uncaught ReferenceError` or should it be handled in the code?

Comment (inez) [inline extensions/wikia/CreatePage/js/CreatePage.js:266]: Such case should not happen - string is always passed to redLinkClick.

Comment (mari-mcmurtrie) [inline extensions/wikia/CreatePage/js/CreatePage.js]: Inez said this is ok.

**Pull request merged**